src/components/Tabs/Tabs.tsx

```diff
@@ -65,29 +65,20 @@ export const Tabs = ({
   onChange,
   ...other
 }: Props) => {
-  /**
-   * Initialize states, constants, and refs
-   */
   const ref = useRef<number | undefined>();
   const headerRef = useRef<HTMLDivElement>(null);
   const [activeIndexState, setActiveIndexState] = useState(activeIndex);
   const [scrollableLeft, setScrollableLeft] = useState<boolean>(false);
   const [scrollableRight, setScrollableRight] = useState<boolean>(false);
 
-  /**
-   * Set the only children components allowed within <Tabs> to be Tab
-   */
-  const tabs = useCallback<() => React.ReactElement<TabProps>[]>(() => {
-    // TODO: apply types from module, specify type definitions in custom.d.ts,
-    // or replace with inline equivalent
+  /** Get all children that are Tab components. Ignore any others. */
+  const getTabs = useCallback(() => {
```

Comment:
Renamed some things for clarity. For example, "tabs" doesn't really tell me this is a function returning the tabs.